README.md

```diff
@@ -11,11 +11,26 @@ repository. Whereas django-calaccess-raw-data supports only state-level data, we
 aim to add support for importing data from local jurisdictions who use Netfile
 to host their data.
 
+## Dependancies
+```
+python, pip, virtualenv
+Python 2.7.9 includes these:
+    Mac: brew install python
+postgresql:
+    Mac: brew install postgresql
+```
+
 ## Installation
 ```
 virtualenv env
 . env/bin/activate
 pip install -r requirements_dev.txt
+## create the database calaccess_raw
+mysql --user root 
+mysql> create database calaccess_raw;
+mysql> \q
```

Comment:
If postgresql is a dependency, shouldn't we be using that as a DB instead of mysql here?